docs/current_docs/api/chaining.mdx

```diff
@@ -126,15 +126,15 @@ To start a `Service` returned by a Dagger Function and have it forward traffic t
 Here is an example of starting an NGINX service on host port 80 by chaining calls to `Container.asService()` and `Service.up()`:
 
 ```shell
-dagger -m github.com/kpenfound/dagger-modules/nginx@v0.1.0 call container as-service up
+dagger -m github.com/kpenfound/dagger-modules/nginx@v0.1.0 -c 'container | as-service | up'
 ```
 
 By default, each port maps to the same port on the host. To specify a different mapping, use the additional `--ports` argument with a list of host/service port mappings. To bind ports randomly, use the `--random` argument.
 
 To start the same service and map NGINX port 80 to host port 8080, use:
 
 ```shell
-dagger -m github.com/kpenfound/dagger-modules/nginx@v0.1.0 call container as-service up --ports=8080:80
+dagger -m github.com/kpenfound/dagger-modules/nginx@v0.1.0 -c 'container | as-service | up --ports=8080:80'
```

Comment:
This gives an error: `Error: unsupported argument of kind map`. This is blocked by https://github.com/dagger/dagger/issues/9579

Reply:
This is fixed in main.